CONTRIBUTING.md

```diff
@@ -175,7 +175,7 @@ call Pyxis API). The way to do this is to create a file with mock shell function
 as the commands you want to mock) and inject this file to the beginning of each `script` field in
 the task step that needs mocking.
 
-For reference implementation, check [push-sbom-to-pyxis/tests/](/tasks/push-sbom-to-pyxis/tests/). Here's a break down of how it's done:
+For reference implementation, check [push-sbom-to-pyxis/tests/](https://github.com/redhat-appstudio/release-service-catalog/tree/fb4002711259c167c9815cc32a9ca9a6f5b065ca/tasks/push-sbom-to-pyxis/tests/). Here's a break down of how it's done:
```

Comment:
We is this changed? We shouldn't map to specific commits as this will go through different environments and keep changing.

Reply:
Oh, because in the change I am _deleting_ the push-sbom-to-pyxis task.

Unfortunately, it was the task that was used as an *example* in CONTRIBUTING.md for how to write a good test.

If I just refer to the directory, it will be blank when users go to visit it.

Alternatively, we could rewrite the docs to use a _different_ task as an example; something other than push-sbom-to-pyxis. But, I wasn't up for that work yet.

Reply:
+1. I think the example should reside in an actual _example task_ that we should maintain.

Reply:
updated text to use an example from another task